packages/ui/package.json

```diff
@@ -3,9 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
+    "dev": "start-storybook -p 6006",
```

Comment:
Adjust script to start a storybook inside readme as well, because it's still `yarn w ui storybook`